Firestore/Example/Tests/Integration/API/FIRWriteBatchTests.mm

```diff
@@ -323,4 +324,54 @@ - (void)testUpdateNestedFields {
   [self awaitExpectations];
 }
 
+// Returns how much memory the test application is currently using, in megabytes (fractional part is
+// truncated), or -1 if the OS call fails.
+// TODO(varconst): move the helper function and the test into a new test target for performance
+// testing.
+long long getCurrentMemoryUsedInMb() {
+  mach_task_basic_info taskInfo;
+  mach_msg_type_number_t taskInfoSize = MACH_TASK_BASIC_INFO_COUNT;
+  const auto errorCode =
+      task_info(mach_task_self(), MACH_TASK_BASIC_INFO, (task_info_t)&taskInfo, &taskInfoSize);
+  if (errorCode == KERN_SUCCESS) {
+    const int bytesInMegabyte = 1000 * 1000;
```

Comment:
nit: 1024 * 1024? not that much difference, up to you.

Reply:
Done. I guess I used the SI definition.